sfa_dash/static/js/report-form-utilities.js

```diff
@@ -1421,3 +1421,20 @@ report_utils.help_popup = function(help_name, help_text){
     var help_box = $(`<span class="${help_name}-help-text form-text text-muted help-text collapse" aria-hidden="">${help_text}</span>`);
     return [help_button, help_box]
 }
+report_utils.register_forecast_fill_method_validator = function(
+    forecast_type='deterministic'){
```

Comment:
maybe make this `''` to avoid potential confusion on the probabilistic form. Or pass the type in all of the register calls.